lively.morphic/world.js

```diff
@@ -202,7 +202,7 @@ export class World extends Morph {
 
   onMouseDown (evt) {
     if (evt.state.menu) evt.state.menu.remove();
-    this.onWindowScroll();
+    // this.onWindowScroll();
```

Comment:
Why? :thinking:

Reply:
This is also a temporary changed that interfered with master component application. I will need to reintroduce that, since its important to check if the scroll of the world is correct every once in a while. We needed this line here in particular for incorrect transform applied to morph that did not take into account the world scroll.